dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -730,49 +728,60 @@ else if(cont.getStructure().getStructureType()==Structure.STRUCTURE_TYPE_HTMLPAG
      * @return Confirmation message
      * @throws Exception
      */
-    public boolean copyFolder ( String inode, String newFolder ) throws Exception {
+    public String copyFolder ( String inode, String newFolder ) throws Exception {
```

Comment:
I think we should throw a Exception in case of error, both methods should return void